tests/integration/golang/fixtures/app.go

```diff
@@ -0,0 +1,80 @@
+package fixtures
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/rotisserie/eris"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/pkg/database"
+)
+
+// AppFixtures represents data fixtures object.
+type AppFixtures struct {
+	baseFixtures
+	*database.DbInstance
+}
+
+// NewAppFixtures creates new instance of AppFixtures.
+func NewAppFixtures(databaseDSN string) (*AppFixtures, error) {
+	db, err := database.ConnectDB(
+		databaseDSN,
+		1*time.Second,
+		20,
+		false,
+		false,
+		"",
+	)
+	if err != nil {
+		return nil, eris.Wrap(err, "error connection to database")
+	}
+	return &AppFixtures{
+		baseFixtures: baseFixtures{db: db.DB},
+		DbInstance:   db,
+	}, nil
+}
+
+// CreateTestApp creates a new test App.
+func (f AppFixtures) CreateTestApp(
```

Comment:
Im not sure that we need a `Test` prefix. Since the place where we use `fixtures` only `integration` tests we always know that we create a `test` records.